proto/tabletmanagerdata.proto

```diff
@@ -541,3 +542,14 @@ message VDiffOptions {
   VDiffCoreOptions core_options = 2;
   VDiffReportOptions report_options = 3;
 }
+
+message UpdateVRWorkflowRequest {
+  string workflow = 1;
+  string cells = 2;
```

Comment:
woudln't it be better to have a `[]string`, ie `repeat string` here?

Reply:
It's currently a CSV string on the command line, a CSV string in flags (could be changed), a CSV string in the RPC (could be changed), and a CSV string in the database. I'm open to changing the the flag handling, and thus the RPC. Did you have specific advantages in mind?

Reply:
> Did you have specific advantages in mind?
The trivial advantage as you might assume I had in mind. If everything else is consistently a `string`, keep it that way.